browser/brave_wallet/DEPS

```diff
@@ -1,3 +1,22 @@
 include_rules = [
-  "+brave/third_party/argon2/src/include/argon2.h",
+  "+brave/components/api_request_helper",
+  "+brave/components/permissions",
+  "+brave/components/version_info",
+  "+brave/components/content_settings/core/browser",
+  "+brave/components/brave_shields/content/test",
+  "+brave/components/ephemeral_storage",
 ]
```

Comment:
Most likely these two 
```
  "+brave/components/brave_shields/content/test",
  "+brave/components/ephemeral_storage",
```
are not needed by wallet

Reply:
I might have mixed them together from something else when splitting them up. Removed